dev-python/pycryptodome/pycryptodome-3.4.3.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2016 Gentoo Foundation
```

Comment:
Adding the new package really needs to go into a separate commit (preceding the one adding the dep).